app/utils/callback/lava.py

```diff
@@ -74,6 +74,8 @@
     models.VCS_COMMIT_KEY: "git.commit",
     models.IMAGE_TYPE_KEY: "image.type",
     models.PLAN_KEY: "test.plan",
+    models.BUILD_ENVIRONMENT_KEY: "job.build_environment",
+    models.FILE_SERVER_RESOURCE_KEY: "job.file_server_resource"
```

Comment:
It's better to keep a trailing comma to avoid having to edit this line later on when adding extra items to the list.